contrib/gitlab-pages/.gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+# Use latest node image
+image: node:latest
+
+# Cache modules
+# http://docs.gitlab.com/ee/ci/yaml/README.html#cache
+cache:
+  paths:
+    - node_modules/
```

Comment:
Caching the `node_modules` is considered bad practice, as it is possible to have issues when upgrading dependency versions. Instead, one should cache the [actual npm cache folder](https://docs.gitlab.com/ee/ci/caching/#cache-nodejs-dependencies).

I have verified my version of the cache to work [here](https://gitlab.com/darekkay/static-marks-gitlab-ci/-/jobs/1111687733).